### DISCUSSION TITLE
Monika After Story Discord i got kicked

### DISCUSSION THREAD

**Discussion by alexifeu** [Monika-After-Story/MonikaModDev#9688] [QA - tech support + general]
I am really sad. Could someone please help me they kicked me. No reason or warning or anything. I didn't do anything bad and didn't harm aynone my discord is Alexifeu#8721 If you can talk to someone I would apreaciate it! :D

I Love the game and the people from the discord :( OFC if something happened I will apologize and do better! :(
![@Reichs Winkler - Discord 27 10 2022 04_09_29](https://user-images.githubusercontent.com/54742509/198174470-3c50b606-25fd-4219-9534-5e697377413b.png)

**Comment by FreeAlexifeu** [comment]
Help my Broddah out he did no wrong 🙏

**Comment by Booplicate** [comment]
GitHub is for mod related issues and discussions only. If you have an issue with the server, contact the staff via discord.
